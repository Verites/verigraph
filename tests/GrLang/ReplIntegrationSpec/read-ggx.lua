rules, names = GrLang.readGGX('grammars/Elevator/elevator.ggx')

print('Types:')
GrLang.print_types()
print()

print('Rules:')
print()
for _, name in ipairs(names) do
  print(name, rules[name])
end