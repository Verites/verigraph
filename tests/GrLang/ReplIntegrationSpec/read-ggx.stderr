Error: ./src/repl/lua/grlang.lua:80: attempt to call a nil value (field 'sort')
